meta-mender-demo/recipes-extended/images/mender-image-full-cmdline-rofs-commercial.bb

```diff
@@ -0,0 +1,12 @@
+require recipes-extended/images/core-image-full-cmdline.bb
+
```

Comment:
Question to both of you, shouldn't this image be in `meta-mender-commercial` layer? It implicitly depends on that layer (adding `mender-binary-delta`).

Reply:
I was unsure about this, when creating it. But I guess you are right. It is probably a better fit!

I'll move it.

Reply:
Oh good point, yes!